scripts/s3-support-bundles/s3_bundle_generate.sh

```diff
@@ -406,17 +415,17 @@ then
 fi
 
 ## Collect LDAP data
+echo "Collecting ldap data"
 mkdir -p $ldap_dir
 if [[ $? != 0 || -z "$rootdnpasswd" ]]
 then
     echo "ERROR: ldap admin password: '$rootdnpasswd' is not correct, skipping collection of ldap data."
 else
     # Run ldap commands
-    ldapsearch -b "cn=config" -x -w "$rootdnpasswd" -D "cn=admin,cn=config" -H ldapi:/// > "$ldap_config"  2>&1
-    ldapsearch -s base -b "cn=subschema" objectclasses -x -w "$rootdnpasswd" -D "cn=admin,dc=seagate,dc=com" -H ldapi:/// > "$ldap_subschema"  2>&1
-    ldapsearch -b "ou=accounts,dc=s3,dc=seagate,dc=com" -x -w "$rootdnpasswd" -D "cn=admin,dc=seagate,dc=com" "objectClass=Account" -H ldapi:/// -LLL ldapentrycount > "$ldap_accounts" 2>&1
-    ldapsearch -b "ou=accounts,dc=s3,dc=seagate,dc=com" -x -w "$rootdnpasswd" -D "cn=admin,dc=seagate,dc=com" "objectClass=iamUser" -H ldapi:/// -LLL ldapentrycount > "$ldap_users"  2>&1
-
+    ldapsearch -b "cn=config" -x -w "$rootdnpasswd" -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldapi:///  > "$ldap_config"  2>&1
```

Comment:
instead of "rootdnpasswd" please change it as "sgiamadminpasswd"

Reply:
Done